app/assets/javascripts/discourse/tests/integration/components/float-kit/d-default-toast-test.js

```diff
@@ -29,7 +29,7 @@ module(
       this.toast = new DToastInstance(this, {});
 
       await render(
-        hbs`<DDefaultToast @data={{this.toast.options.data}} @showProgressBar={{true}} @autoClose={{true}} @onRegisterProgressBar={{(noop)}} />`
+        hbs`<DDefaultToast @data={{this.toast.options.data}} @showProgressBar={{true}} @onRegisterProgressBar={{(noop)}} />`
```

Comment:
Hmm I guess it's hard to check the `autoclose`. 🤔 

Btw, is the progress bar transition time configurable? e.g. if it's currently 1 second to close, can we set this value programmatically?

Reply:
I probably should have left a comment, autoclose it is no longer relevant here as it was removed in #26537 

It's actually based on the duration (ms), so if the option is set to `duration: 1000` it closes the toast within 1 second, and the progress bar matches this time too automatically.